AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/Base/AsyncExpectation.swift

```diff
@@ -1,130 +0,0 @@
-//
-// Copyright Amazon.com Inc. or its affiliates.
-// All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-
-import Foundation
-import XCTest
-import Amplify
-
-public actor AsyncExpectation {
```

Comment:
by removing this, how does APIHostApp and the rest of the integration test targets pull in AsyncExpectation?